src/main/java/com/iexec/sms/secret/web2/Web2Secrets.java

```diff
@@ -17,20 +17,18 @@
 package com.iexec.sms.secret.web2;
 
 import com.iexec.sms.secret.Secret;
-import lombok.Data;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
 import org.hibernate.annotations.GenericGenerator;
 
 import javax.persistence.*;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
+import java.util.stream.Collectors;
 
-@Data
-@Getter
 @Entity
-@NoArgsConstructor
+@Getter
+@Builder
```

Comment:
Why do you add this annotation if you don't use it ?

Reply:
I planned to use it... But finally removed its usages. So annotation can be removed. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/134/commits/0387d46e55e63b64dfef293b2f07d9f8cb715961.